### PR TITLE
Update cinema.json

### PR DESCRIPTION
**Pull request by dan13l9**
I made a few small corrections and additions. Can i add new brands with this method? Thank you.

### CODE REVIEW

File: data/brands/amenity/cinema.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
       }
     },
     {
-      "displayName": "AMC",
+      "displayName":",
```
Comment (arch0345) [inline data/brands/amenity/cinema.json:44]: Looks like you accidentally removed the displayName from AMC
```suggestion
      "displayName": "AMC",
```

File: data/brands/amenity/cinema.json [hunk 2]
```diff
@@ -144,14 +144,15 @@
...
-        "include": ["cz", "hu", "pl", "ro", "sk"]
+        "include": ["bg, "cz", "hu", "pl", "ro", "sk"]
       },
       "tags": {
         "amenity": "cinema",
         "brand": "Cinema City",
         "brand:wikidata": "Q543651",
         "brand:wikipedia": "en:Cinema City International",
         "name": "Cinema City"
+        "operator": "Cineworld"
```
Comment (arch0345) [inline data/brands/amenity/cinema.json:155]: Their Wikipedia article suggests that Cineworld is just the parent company and that Cinema City operates the cinemas
```suggestion
```

File: data/brands/amenity/cinema.json [hunk 3]
```diff
@@ -546,6 +547,7 @@
         "brand:wikidata": "Q6484805",
         "brand:wikipedia": "en:Landmark Theatres",
         "name": "Landmark Theatres",
+        "operator: "Kinepolis",
```
Comment (arch0345) [inline data/brands/amenity/cinema.json:550]: I think you mixed [Landmark Theatres](https://en.wikipedia.org/wiki/Landmark_Theatres) with [Landmark Cinemas](https://en.wikipedia.org/wiki/Landmark_Cinemas). It looks like Landmark Cinemas isn't in NSI yet so feel free to add it
```suggestion
```

**Review by arch0345 — CHANGES_REQUESTED** [review]
Thanks and welcome! I've left some suggestions that should be reviewed before this is merged.

File: data/brands/amenity/cinema.json [hunk 4]
```diff
@@ -144,14 +144,15 @@
       "displayName": "Cinema City",
       "id": "cinemacity-26c608",
       "locationSet": {
-        "include": ["cz", "hu", "pl", "ro", "sk"]
+        "include": ["bg, "cz", "hu", "pl", "ro", "sk"]
```
Comment (LaoshuBaby) [inline data/brands/amenity/cinema.json:147]:
```suggestion
        "include": ["bg", "cz", "hu", "pl", "ro", "sk"]
```

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/amenity/cinema.json [hunk 5]
```diff
@@ -184,7 +184,7 @@
     {
       "displayName": "Cinemax",
       "id": "cinemax-4c7ab3",
-      "locationSet": {"include": ["sk"]},
+      "locationSet": {"include": ["ro, "sk"]},
```
Comment (arch0345) [inline data/brands/amenity/cinema.json:187]:
```suggestion
      "locationSet": {"include": ["ro", "sk"]},
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

File: data/brands/amenity/cinema.json [hunk 6]
```diff
@@ -702,6 +702,7 @@
         "brand:wikidata": "Q1144802",
         "brand:wikipedia": "pl:Multikino",
         "name": "Multikino"
+        "operator": "Vue Cinemas"
```
Comment (arch0345) [inline data/brands/amenity/cinema.json:705]:
```suggestion
        "operator": "Vue Cinemas",
```

**Review by arch0345 — CHANGES_REQUESTED** [review]

**Pull request merged**